react/components/FormattedPrice.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react'
+import { FormattedNumber } from 'react-intl'
+
+import { useCurrency } from './CurrencyContext'
+
+interface Props {
+  value: number
+}
+
+const FormattedPrice: FC<Props> = ({ value }) => {
+  const currency = useCurrency()
```

Comment:
just for consistency of previous files

```suggestion 
  const currency = useCurrency()
  
```

Reply:
💅